include/envoy/tls/connection.h

```diff
@@ -6,7 +6,7 @@
 #include "envoy/common/pure.h"
 
 namespace Envoy {
-namespace Ssl {
+namespace Tls {
 
 /**
  * Base connection interface for all SSL connections.
```

Comment:
TLS

Reply:
I'm not sure what you mean by these comments. I moved include/envoy/ssl and Envoy::Ssl to include/envoy/tls and Envoy::Tls::

Thanks